remote/client-inject.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const injectInterval = setInterval(() => {
+    if (!client) return;
+    client.OPEN_TIMEOUT = 100_000;
```

Comment:
```suggestion
    client.OPEN_TIMEOUT = 30_000;
```

100 seconds might be bit too long. If think it's sufficient to time out after 30 seconds.